pkg/api/message/newest_envelope_test.go

```diff
@@ -62,21 +63,24 @@ func parseResults(
 
 func TestGetNewestEnvelope(t *testing.T) {
 	api, db, _ := apiTestUtils.NewTestReplicationAPIClient(t)
-	querier := queries.New(db)
-
 	installationID1 := testutils.RandomGroupID()
 	installationID2 := testutils.RandomGroupID()
 	installationID3 := testutils.RandomGroupID()
 	installationID4 := testutils.RandomGroupID()
 
 	// Installation ID 1 has three key packages
-	topic1, _ := writeKeyPackage(t, querier, installationID1[:])
+	topic1, discared := writeKeyPackage(t, db, installationID1[:])
```

Comment:
Isn't the second return value the sequenceId?

Reply:
probably leftover from debugging